great_expectations/data_context/types/base.py

```diff
@@ -2410,7 +2410,7 @@ def __str__(self) -> str:
         return self.__repr__()
 
 
-class CheckpointValidationConfig(AbstractConfig):
+class CheckpointValidationDefinition(AbstractConfig):
```

Comment:
Don't think we're actually going to use this class anymore but I'm okay with the change